01-node-tutorial/answers/02-globals.js

```diff
@@ -0,0 +1,6 @@
+
+console.log(process.env.MY_VAR)
+
+const glob = ['Directory Name: ' + __dirname, 'File Name: ' + __filename]
+
+module.exports = glob
```

Comment:
This program was supposed to log the global variables to the console. You accessed the variables correctly but you exported them in an array instead of logging them to the console.

Reply:
Actually, I just saw that you used the main program from question 3 to import those values and print them. That's awesome! You can ignore the comment above.

Reply:
Thank you for your feedback!